.github/workflows/publish-e2e-containers.yaml

```
@@ -0,0 +1,87 @@
+  # Enable BuildKit for cache mounts
+  DOCKER_BUILDKIT: 1
+  BUILDKIT_PROGRESS: plain
+
+jobs:
+
+
+
+  publish-flightctl-e2e-containers:
+    strategy:
+      matrix:
+        image: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)